lab-isaiah/src/component/dashboard-container/index.js

```diff
@@ -0,0 +1,60 @@
+import './_dashboard-container.scss';
+import React from 'react';
+import {connect} from 'react-redux';
+import {
+  categoryCreate,
+  categoryUpdate,
+  categoryDelete,
+} from '../../action/category-actions';
+import CategoryForm from '../category-form';
+import CategoryItem from '../category-item';
+
+class DashboardContainer extends React.Component {
```

Comment:
please don't copy and paste demo code, it's a different app completely and it will break your app.